conversant/prompt_chatbot.py

```diff
@@ -290,6 +375,26 @@ def to_dict(self) -> Dict[str, Any]:
             "latest_prompt": self.latest_prompt,
         }
 
+    def _check_prompt_size(self) -> None:
+
+        self.start_prompt_size = self.co.tokenize(self.get_current_prompt(None)).length
```

Comment:
```suggestion
        self.start_prompt_size = self.co.tokenize(self.prompt.to_string()).length
```
Calling `get_current_prompt(None)` here leads to this extra interaction:
"""
User: None
Bot: 
"""